docs/search/full-text/index.mdx

```diff
@@ -291,6 +291,9 @@ The following tokenizer names are accepted by the `paradedb.tokenizer` function.
 <ParamField body="lowercase">
   Equivalent to `raw`, but also converts to lowercase.
 </ParamField>
+<ParamField body="regex">
+  Tokenizes text using a regular expression. The regular expression can be specified with the `pattern` parameter.
```

Comment:
nit: Please document the `pattern` parameter as an expandable config option. See `ngram` or `stem` docs.

Reply:
Thank you. I have added that in the docs.